samples/mpu_test/prj_allow_flash_write_mcux.conf

```diff
@@ -0,0 +1,3 @@
+CONFIG_CONSOLE_SHELL=y
+CONFIG_FLASH=y
+CONFIG_SOC_FLASH_MCUX=y
```

Comment:
You don't need to set CONFIG_SOC_FLASH_MCUX=y explicitly. The Kinetis SoC defconfigs enable it if CONFIG_FLASH=y